cmake/FindDPCPP.cmake

```diff
@@ -57,6 +57,10 @@ if(NOT "${DPCPP_SYCL_ARCH}" STREQUAL "")
   endif()
 endif()
 
+if("${DPCPP_SYCL_TARGET}" STREQUAL "intel_gpu_pvc")
+  list(APPEND DPCPP_FLAGS "-Xspirv-translator;--spirv-ext=+SPV_INTEL_split_barrier")
```

Comment:
I'm getting this warning `warning: argument unused during compilation: '-Xspirv-translator --spirv-ext=+SPV_INTEL_split_barrier'`. Are these flags actually needed?

Reply:
This is required for the split barrier introduced in `cutlass/arch/barrier.h` file.